data/random-teams.ts

```diff
@@ -2080,6 +2080,7 @@ export class RandomTeams {
 		const pokemonPool = [];
 		for (let species of this.dex.species.all()) {
 			if (species.gen > this.gen || exclude.includes(species.id)) continue;
+			if (this.dex.currentMod === 'gen8bdsp' && species.gen > 4) continue;
```

Comment:
Wouldn't it be better to do `delete species.randomBattleMoves` and `delete species.randomBattleMoves` in `dex-species.ts` for the bdsp mod instead of adding this hardcode in?

Reply:
Maybe but I'd prefer to reduce the amount of hardcodes in the dex files, and this file already has other gen8bdsp hardcodes

Reply:
Does `Dex.species.get` with a BDSP-modded Dex still pull `randomBattleMoves` from the non-BDSP formats-data? It seems like it shouldn't (and if it doesn't, this hardcode wouldn't be necessary).

If BDSP `dex` _does_ inherit SwSh `randomBattleMoves`, then yes, I'd rather have the hardcode in this file (since it's specific to random battles). That might lead to code duplication in `/randbats`, though, which is why I think getting Dex to read only the modded randbats moves data is the better solution.